src/main/surfrdMod.F90

```diff
@@ -518,16 +539,120 @@ subroutine surfrd_cftformat( ncid, begg, endg, wt_cft, fert_cft, cftsize, natpft
        irrig_method = irrig_method_unset
     end if
 
+    ! FATES should over-write these weights later on
     allocate( array2D(begg:endg,1:natpft_size) )
     call ncd_io(ncid=ncid, varname='PCT_NAT_PFT', flag='read', data=array2D, &
          dim1name=grlnd, readvar=readvar)
     if (.not. readvar) call endrun( msg=' ERROR: PCT_NAT_PFT NOT on surfdata file'//errMsg(sourcefile, __LINE__))
     wt_nat_patch(begg:,natpft_lb:natpft_size-1+natpft_lb) = array2D(begg:,:)
     deallocate( array2D )
+ 
 
   end subroutine surfrd_cftformat
 
-!-----------------------------------------------------------------------
+  subroutine surfrd_wtfates( ncid, begg, endg )
```

Comment:
This is the part that I'm struggling with as I'd like to see a solution that would also work when we go away from requiring FATES to have create_crop_landunit=FALSE. I think this needs to be broken up into a couple of parts. One part has to do with the create_crop_landunit, and another part has to do with the setting of FATES PFT's to equal weights. I think this is also where the equalization of FATES patches should go rather than in InitGridCells.

Reply:
We talked about this and it's not clear how it needs to work. So we plan to bring it in as is for now, but do some refactoring later.